### PR TITLE
Add doc about cronjob-kicker schedule

### PR DESCRIPTION
**Pull request by bklei**
## Summary and Scope

Add info about verifying the cronjob-kicker schedule if cronjobs not running.

## Issues and Related PRs

* Resolves [CASMPET-5184](https://jira-pro.its.hpecorp.net:8443/browse/CASMPET-5184)

## Testing

N/A

### Tested on:

N/A

### Test description:

N/A

## Risks and Mitigations

Low

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/kubernetes/CronJobs.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Kubernetes CronJobs
+
+Kubernetes CronJobs create Kubernetes jobs on a repeating schedule, specified in traditional cron syntax.
+
+### CronJobs Not Scheduled
+
+In some cases CronJobs can fail to get scheduled (such as the cray-dns-unbound-manager job) if a previously run job didn't finish properly.  In order to alleviate this condition, CSM deploys a traditional cronjob on one of the master NCNs to perform cleanup such that jobs continue to get rescheduled.
+
+In the event jobs aren't running, ensure the cronjob is specified properly using the following steps.
```
Comment (nrockershousen) [inline operations/kubernetes/CronJobs.md:9]:
```suggestion
In the event jobs aren't running, ensure the CronJob is specified properly using the following steps.
```

File: operations/kubernetes/CronJobs.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Kubernetes CronJobs
```
Comment (nrockershousen) [inline operations/kubernetes/CronJobs.md:1]: Add an entry for this new file to the /operations/index.md file

File: operations/kubernetes/CronJobs.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Kubernetes CronJobs create Kubernetes jobs on a repeating schedule, specified in traditional cron syntax.
+
+### CronJobs Not Scheduled
+
+In some cases CronJobs can fail to get scheduled (such as the cray-dns-unbound-manager job) if a previously run job didn't finish properly.  In order to alleviate this condition, CSM deploys a traditional cronjob on one of the master NCNs to perform cleanup such that jobs continue to get rescheduled.
```
Comment (nrockershousen) [inline operations/kubernetes/CronJobs.md:7]:
```suggestion
In some cases, CronJobs can fail to get scheduled (such as the cray-dns-unbound-manager job) if a previously run job did not finish properly. In order to alleviate this condition, CSM deploys a traditional CronJob on one of the master NCNs to perform cleanup such that jobs continue to get rescheduled.
```

File: operations/kubernetes/CronJobs.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+In the event jobs aren't running, ensure the cronjob is specified properly using the following steps.
+
+1. Determine the master NCN hosting the cronjob-kicker (typically ncn-m001 or ncn-m002):
```
Comment (nrockershousen) [inline operations/kubernetes/CronJobs.md:11]:
```suggestion
1. Determine the master NCN hosting the cronjob-kicker (typically `ncn-m001` or `ncn-m002`):
```

File: operations/kubernetes/CronJobs.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+   ```bash
+   ncn-m001# ls  /etc/cron.d/cronjob-kicker
+   /etc/cron.d/cronjob-kicker
+   ```
+
+1. Verify the crontab schedule.
+
+   This cronjob is intended to run once every two hours.  However, in some CSM 1.x
```
Comment (nrockershousen) [inline operations/kubernetes/CronJobs.md:20]:
```suggestion
   This CronJob is intended to run once every two hours. However, in some CSM 1.x
```

File: operations/kubernetes/CronJobs.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+   releases, the cronjob schedule can run too frequently, preventing jobs from being
```
Comment (nrockershousen) [inline operations/kubernetes/CronJobs.md:21]:
```suggestion
   releases, the CronJob schedule can run too frequently, preventing jobs from being
```

File: operations/kubernetes/CronJobs.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+   properly scheuled.  Ensure the schedule looks as follows:
+
+   ```bash
+   ncn-m001# cat /etc/cron.d/cronjob-kicker
+   0 */2 * * * root KUBECONFIG=/etc/kubernetes/admin.conf /usr/bin/cronjob_kicker.py
+   ```
+
+   `NOTE:` Ensure the first character is `0` and not `*`, which will cause the cronjob
```
Comment (nrockershousen) [inline operations/kubernetes/CronJobs.md:29]:
```suggestion
   > **NOTE:** Ensure the first character is `0` and not `*`, which will cause the CronJob
```

File: operations/kubernetes/CronJobs.md (hunk 2 above)
Comment (bklei) [reply operations/kubernetes/CronJobs.md:1]: Will do, thanks for the review!

Comment (bklei) [reply operations/kubernetes/CronJobs.md:1]: added

**Review by nrockershousen — APPROVED** [review]

**Review by rgross727 — APPROVED** [review]

**Review by alexanderkingh — APPROVED** [review]

**Pull request merged**